Textor/Controller/SettingsViewController.swift

```diff
@@ -48,11 +55,22 @@ class SettingsViewController: UITableViewController {
 	}
 
 	@IBAction func close(_ sender: UIBarButtonItem) {
-		
 		self.dismiss(animated: true, completion: nil)
 		
 	}
-	
+    
+    @IBAction func fontSizeChanged(_ sender: UIStepper) {
+        UserDefaults.standard.set(sender.value, forKey: "fontSize")
+        fontSizeLabel.text = "\(Int(sender.value))"
+    }
+    
+    @IBAction func themeChanged(_ sender: UISwitch) {
+        UserDefaults.standard.set(sender.isOn, forKey: "darkMode")
+        let alert = UIAlertController(title: "Theme updated!", message: "Restart the app for changes to take effect", preferredStyle: .alert)
```

Comment:
I'm not really a fan of "Restart the app for changes to take effect". Ideally the app would change the theme without restart.